README.md

```diff
@@ -1,67 +1,146 @@
-2. set up music bot capabilites with queueing, pausing, stopping, etc. 
-    - potentially take all music bot code and put into a separate bot so as to avoid voice-overlap. alternatively, have the bot not do anything voice related while playing audio/music.
-3. set audio recording capabilities
-4. add constants.js with constant definitions
-5. change /playaudio and /listaudio to /audio play and /audio list subcommands
\ No newline at end of file
+# PASTRAMI BOT
+
+[![GitHub issues](https://img.shields.io/github/issues/jacobc18/js-discord-bot)](https://github.com/jacobc18/js-discord-bot/issues)
+
+![hello there](https://media.giphy.com/media/3ornk57KwDXf81rjWM/giphy.gif)
+
+
+
+
+Pastrami is a multifunctional JavaScript Discord bot with abilities including
+
+- TTS greetings for voice channel user join
+- Audio streaming via YouTube links 
+- Information on famous bald people
+
+## Installation
+
+Requires [NodeJS v16.x.x](https://nodejs.org/en/) or higher and the latest version of [NPM](https://docs.npmjs.com/cli/v7/commands/npm-install)
+
+### Windows
+
+```
+$ windows_setup.bat
+```
+
+### Manual
+
+In package directory run
+
+```
+$ npm install
+```
+
+## Setup
+
+In the repo folder create a file called ``` .env ``` with the contents
+
+```
+DISCORD_TOKEN=your-discord-bot-token-goes-here
+CLIENT_ID=your-discord-client-id-goes-here
+```
+
+To get a Discord bot token and Discord client ID see [How to Make a Discord Bot in the Developer Portal](https://realpython.com/how-to-make-a-discord-bot-python/#how-to-make-a-discord-bot-in-the-developer-portal).
+
+We utilize [say.js](https://github.com/Marak/say.js) to convert text strings to mp3 files. Say.js uses some [FFMpeg](https://www.ffmpeg.org/) libraries to operate, so you will need to install the most up to date version. You can learn how to do that [here](https://www.wikihow.com/Install-FFmpeg-on-Windows). 
+
+(Note: say.js uses [Festival](https://www.cstr.ed.ac.uk/projects/festival/) to operate on Linux, so check their [README](https://github.com/Marak/say.js/blob/master/README.md) to get that squared away)
+
+If you want to utilize Pastrami's audio streaming capabilities for non YouTube content, you will need to populate ``` /data/audiofiles``` with the mp3/mp4 files you would like to play.
+
+## Usage
+
+### Startup 
+
+To start the bot locally on your machine run
+
+```
+$ node index.js
+```
+
+This will also connect the bot to any guilds you have linked it to via [Discord's Developer Portal](http://discordapp.com/developers/applications).
+
+### Greetings
+
+To customize greetings navigate to ``` /data/greetings.json```. 
+
+- Default Greetings
+
+```json
+"default": {
+        "greetings": [
+            "welcome *NAME*",
+            "say hi to *NAME*"
+        ]
+    },
+```
+
+In ``` greetings.json```  the \*NAME\* variable is used to reference the server nickname of the user who has joined the voice channel. 
+
+- Guild Default Greetings 

```

Comment:
Let's comment out the 'Guild Default Greetings' for now. Currently the code uses a special string to do this ONLY for the JFEP Guild and would not work for any other guild. I'll be updating that in the future and at that time I can uncomment this section. Having this here right now is inaccurate